extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -178,6 +257,12 @@ export class AssessmentDetailsHeader {
 			assessmentHeaderValues.push({
 				value: configurationValue
 			})
+
+			if (configurationValue !== "--") {
+				await this._viewDetailsLink.updateCssStyles({ 'display': 'block' });
```

Comment:
we can have this.linksContainer.updateCssStyles({ 'display': 'block' }); instead of changing for three?